include/CFL/CFLAlias.h

```diff
@@ -61,16 +62,17 @@ class CFLAlias : public PointerAnalysis
         GrammarBuilder gReader = GrammarBuilder(Options::GrammarFilename);
         CFGNormalizer normalizer = CFGNormalizer();
         CFLGraphBuilder cflGraphBuilder = CFLGraphBuilder();
+        CFLChecker cflChecker = CFLChecker();
         /// Assume Read From Const Graph, associate label symbol is hard coded
         if (Options::GraphIsFromDot == false)
         {
             PointerAnalysis::initialize();
             Map<std::string, SVF::CFLGraph::Symbol> ConstMap =  {{"Addr",0}, {"Copy", 1},{"Store", 2},{"Load", 3},{"Gep_i", 4},{"Vgep", 5},{"Addrbar",6}, {"Copybar", 7},{"Storebar", 8},{"Loadbar", 9},{"Gepbar_i", 10},{"Vgepbar", 11}};
             GrammarBase *generalGrammar = gReader.build(&ConstMap);
             ConstraintGraph *consCG = new ConstraintGraph(svfir);
-            graph = cflGraphBuilder.buildBigraph(consCG, &generalGrammar->terminals, &generalGrammar->nonterminals);
-            grammar = normalizer.normalize(generalGrammar, &graph->kind2AttrMap);
-            graph->setMap(&grammar->terminals, &grammar->nonterminals);  // In normalization might generate more intermediate variable
+            graph = cflGraphBuilder.buildBigraph(consCG, generalGrammar);
```

Comment:
why do we need `generalGrammar ` to build a graph if we have a checker afterwards?